packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -117,29 +118,85 @@ class _AllocationTracingTableState extends State<AllocationTracingTable> {
 
   @override
   Widget build(BuildContext context) {
-    return DualValueListenableBuilder<bool, List<TracedClass>>(
-      firstListenable: widget.controller.refreshing,
-      secondListenable: widget.controller.classList,
-      builder: (context, _, classList, __) {
-        return FlatTable<TracedClass>(
-          columns: columns,
-          data: classList,
-          keyFactory: (e) => Key(e.cls.id!),
-          onItemSelected: widget.controller.selectTracedClass,
-          sortColumn: _checkboxColumn,
-          secondarySortColumn: secondarySortColumn,
-          sortDirection: sortDirection,
-          selectionNotifier: widget.controller.selectedTracedClass,
-          onSortChanged: (column, direction, {secondarySortColumn}) {
-            // Keep track of sorting state so it doesn't get reset when
-            // `controller.refreshing` changes.
-            setState(() {
-              sortDirection = direction;
-              secondarySortColumn = secondarySortColumn;
-            });
+    return Column(
+      children: [
+        Padding(
+          padding: const EdgeInsets.only(bottom: densePadding),
+          child: ClassFilterTextField(
+            controller: widget.controller,
+          ),
+        ),
+        DualValueListenableBuilder<bool, List<TracedClass>>(
+          firstListenable: widget.controller.refreshing,
+          secondListenable: widget.controller.classList,
+          builder: (context, _, classList, __) {
+            return Expanded(
```

Comment:
put the Expanded widget around DualValueListenableBuilder so that it is clear that the Expanded is a direct child of the Column